management-api-agent-common/src/main/java/io/k8ssandra/metrics/config/ConfigReader.java

```diff
@@ -0,0 +1,34 @@
+package io.k8ssandra.metrics.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ConfigReader {
+    public static final String CONFIG_PATH_PROPERTY = "collector-config-path";
+    public static final String CONFIG_PATH_DEFAULT = "/opt/management-api/configs/metric-collector.yaml";
+
+    public static Configuration readConfig() {
+        // Check env variable if there's any changes to the config path
+        String configPath = System.getProperty(CONFIG_PATH_PROPERTY);
+        if(configPath == null) {
+            configPath = CONFIG_PATH_DEFAULT;
+        }
```

Comment:
nit: could be refactored into a ternary

Reply:
Those are ugly and hard to read. I only use them when I want different bytecode to be emitted ;)